kratos/python/add_mapping_variables_to_python.cpp

```diff
@@ -0,0 +1,54 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrándiz
+//
+
+// System includes
+
+// External includes
+#include <boost/python.hpp>
+
+// Project includes
+#include "includes/define.h"
+#include "includes/mapping_variables.h"
+#include "python/add_mapping_variables_to_python.h"
+
+#ifdef KRATOS_REGISTER_IN_PYTHON_IN_PYTHON_VARIABLE
```

Comment:
Dosn't seem to do anything to be honest. Also, I spoke with @pooyan-dadvand and it looks that this some kind of bad replace ( name is veeeery susoicios ).

The macro also appears in:
- add_ale_variables_to_python.cpp
- add_cfd_variables_to_python.cpp
- add_legacy_structural_variables_to_python.cpp

And also does nothing. I would say is safe to remove, and also create an issue to remove it from the other files as well.